backend/routes/api/export.js

```diff
@@ -75,7 +75,10 @@ router.get(
 
         order: [[Sequelize.col("operatingYear"), "ASC"]],
       })
-    ).map((year) => year.operatingYear);
+    ).map((year) => ({
+      label: year.operatingYear.toString(),
+      value: year.operatingYear,
+    }));
 
```

Comment:
I needed to convert an array of numbers to an array of objects for the react-select options prop.